homeassistant/components/life360/translations/bg.json

```diff
@@ -5,13 +5,9 @@
             "invalid_auth": "\u041d\u0435\u0432\u0430\u043b\u0438\u0434\u043d\u043e \u0443\u0434\u043e\u0441\u0442\u043e\u0432\u0435\u0440\u044f\u0432\u0430\u043d\u0435",
             "reauth_successful": "\u041f\u043e\u0432\u0442\u043e\u0440\u043d\u043e\u0442\u043e \u0443\u0434\u043e\u0441\u0442\u043e\u0432\u0435\u0440\u044f\u0432\u0430\u043d\u0435 \u0431\u0435\u0448\u0435 \u0443\u0441\u043f\u0435\u0448\u043d\u043e"
         },
-        "create_entry": {
```

Comment:
Other translation files should not be touched. It also has no use, as they are overwritten each night.

Reply:
It says you requested a change, but I can't tell what change you've requested.

The main problem is entries in strings.json that have previously been removed are still showing up in the files in the translations folder. If the process that overwrites these files every night is not removing those entries from the files it creates, how can I get them removed from those files?

E.g., in [this change](https://github.com/home-assistant/core/commit/0a65f53356e124592cae37ea1f1873b789e0726b#diff-396f6e9cd80325787b1ede24a06519c2ccbf10c73786aa518e29ce79f31b793b) you can see that the following were removed from translations/en.json (at the same time they were removed from strings.json):
```
config.abort.unknown
config.create_entry
config.error.invalid_username
config.step.user.description
```
But then later that same day a GitHub Action w/ comment ["[ci skip] Translation update"](https://github.com/home-assistant/core/commit/721741281ec1f71f97771995033568b5505c0eb4#diff-396f6e9cd80325787b1ede24a06519c2ccbf10c73786aa518e29ce79f31b793b) added the entries back in.

Why is that process adding entries to the files in the translations folder that aren't in strings.json? Did I do something wrong? Or is that process flawed in some way? But mainly, how can this be fixed, because the entries that should have been removed have placeholders that are not being supplied by the code anymore, which is resulting in errors in the UI on the Integrations page?

Reply:
So how can I fix the problem where strings are getting into the files in the translation folder that aren't in strings.json?